src/xcode/ENA/ENA/Source/Scenes/Events/Cells/EventTableViewCell.swift

```diff
@@ -89,6 +89,8 @@ class EventTableViewCell: UITableViewCell {
 
 		titleLabel.text = cellModel.title
 		addressLabel.text = cellModel.address
+		addressLabel.accessibilityLabel = String(format: AppStrings.TraceLocations.Configuration.address, cellModel.address)
```

Comment:
Please put this in the cell model, as this cell is used for checkins and for QR codes. With this implementation it would also read it for checkins.

Also another remark, maybe check with Oliver Keim again: I think it would make more sense to read "QR-Code für ...." before the title and not after the title and before the address.

Reply:
He has another ticket for that. He wants us to add "Check in" <item details> for title. 
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6389

Reply:
I see that you grabbed that one as well and fix in with the other one, nice!